### ISSUE TITLE
[BUG] vrdashboard is vertically flipped

### ISSUE THREAD

**Issue by apcolvin** [ValveSoftware/SteamVR-for-Linux#615]
**Describe the bug**
New SteamVR display is upside down
![Screenshot_20230926_115109](https://github.com/ValveSoftware/SteamVR-for-Linux/assets/43722087/62ff9942-88a7-4b3b-99d6-bef7a199e500)

**To Reproduce**
Start SteamVR without Steamtours

**Expected behavior**

**System Information (please complete the following information):**
- Distribution: Opensuse tumbleweed
- SteamVR version: 2.0.1
- Steam client version: 1695679261 
- Opted into Steam client beta?: Yes
- Graphics driver version:  Mesa 23.1.7
- Gist for SteamVR System Information: https://gist.github.com/apcolvin/2220ebee202deea818acf44afd49f37b#file-steamvr

**Screenshots**
see above

**Note:** Commenters who are also experiencing this issue are encouraged to include the "System Information" section in their replies.

**Comment by caseif** [comment]
This behavior occurs for me as well, although opting out of the Steam client beta seems to resolve it. Of note, I'm currently experiencing #616 so this was observed via the desktop VR view rather than on the headset display.

Tangentially, I'm growing increasingly frustrated with the number of obvious, universal, breaking bugs present in beta versions of SteamVR, some of which have made it into stable versions. I don't mean to be nasty, but it really does appear that these releases aren't even being launched on a Linux system internally prior to being shipped given how blatant these problems are.

**Comment by TheSHEEEP** [comment]
Same, same.
It's quite something to look at.

My system information:
```Computer Information:
Manufacturer: Micro-Star International Co., Ltd.
Model: MAG B550 TOMAHAWK (MS-7C91)
Form Factor: Desktop
No Touch Input Detected
Processor Information:
CPU Vendor: AuthenticAMD
CPU Brand: AMD Ryzen 9 5900X 12-Core Processor
CPU Family: 0x19
CPU Model: 0x21
CPU Stepping: 0x0
CPU Type: 0x0
Speed: 4950 MHz
24 logical processors
12 physical processors
Hyper-threading: Supported
FCMOV: Supported
SSE2: Supported
SSE3: Supported
SSSE3: Supported
SSE4a: Supported
SSE41: Supported
SSE42: Supported
AES: Supported
AVX: Supported
AVX2: Supported
AVX512F: Unsupported
AVX512PF: Unsupported
AVX512ER: Unsupported
AVX512CD: Unsupported
AVX512VNNI: Unsupported
SHA: Supported
CMPXCHG16B: Supported
LAHF/SAHF: Supported
PrefetchW: Unsupported
Operating System Version:
"Manjaro Linux" (64 bit)
Kernel Name: Linux
Kernel Version: 6.1.53-1-MANJARO
X Server Vendor: The X.Org Foundation
X Server Release: 12101008
X Window Manager: KWin
Steam Runtime Version: steam-runtime_0.20230919.60656
Video Card:
Driver: AMD AMD Radeon RX 6800 XT (navi21, LLVM 16.0.6, DRM 3.49, 6.1.53-1-MANJARO)
Driver Version: 4.6 (Compatibility Profile) Mesa 23.1.7-1
OpenGL Version: 4.6
Desktop Color Depth: 24 bits per pixel
Monitor Refresh Rate: 60 Hz
VendorID: 0x1002
DeviceID: 0x73bf
Revision Not Detected
Number of Monitors: 2
Number of Logical Video Cards: 1
Primary Display Resolution: 1920 x 1080
Desktop Resolution: 3840 x 1221
Primary Display Size: 20.91" x 11.77" (23.98" diag), 53.1cm x 29.9cm (60.9cm diag)
Primary VRAM: 16384 MB
Sound card:
Audio device: ATI R6xx HDMI
Memory:
RAM: 32016 Mb
VR Hardware:
Headset: Valve Index (lighthouse)
Base station or sensor: Valve Corporation Valve SR Imp (lighthouse)
Base station or sensor: Valve Corporation Valve SR Imp (lighthouse)
Controller: Valve Knuckles Right (lighthouse)
Controller: Valve Knuckles Left (lighthouse)
Miscellaneous:
UI Language: English
LANG: en_DE.UTF-8
Total Hard Disk Space Available: 1841908 MB
Largest Free Hard Disk Block: 1016132 MB
Storage:
Number of SSDs: 2
SSD sizes: 2000G,2000G
Number of HDDs: 0
Number of removable drives: 0```

**Comment by TTimo** [comment]
The Y-axis flip should be fixed in latest Steam client beta + SteamVR beta. Are the color channels still swapped? (e.g. background looking brownish)

**Comment by caseif** [comment]
@TTimo Both the y-axis inversion and color swap issues seem to be fixed with 2.0.3 on my system.

**Comment by TTimo** [comment]
thanks!
